testrpc/wait.go

```diff
@@ -9,10 +9,10 @@ import (
 
 type rpcFn func(string, interface{}, interface{}) error
 
-func WaitForLeader(t *testing.T, rpc rpcFn, dc string) {
+// WaitForLeader ensures we have a leader and a node registration.
+func WaitForLeader(t *testing.T, rpc rpcFn, dc, node string) {
```

Comment:
This looks like maybe a leftover (unintentional) change - changing this function signature would affect a lot of tests and downstream things that pull in this package to test with, like consul-template and nomad.

Reply:
Good catch, thanks!